deploy/cdk/lib/manifest-pipeline/deployment-pipeline.ts

```diff
@@ -134,6 +135,23 @@ export class DeploymentPipelineStack extends cdk.Stack {
           cdk.Fn.sub('arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:*'),
         ],
       }))
+      // all the user to create a backup
+      cdkDeploy.project.addToRolePolicy(new PolicyStatement({
+        actions: [
+          'backup:CreateBackupPlan',
+          'backup:CreateBackupSelection',
+          'backup:CreateBackupVault',
+          'backup:DeleteBackupPlan',
+          'backup:DeleteBackupVault',
+          'backup:DeleteBackupVault',
+        ],
+        resources: [
+          cdk.Fn.sub('arn:aws:states:${AWS::Region}:${AWS::AccountId}:backup-plan:*'),
+          cdk.Fn.sub('arn:aws:states:${AWS::Region}:${AWS::AccountId}:backup-vault:*'),
+          cdk.Fn.sub('arn:aws:states:${AWS::Region}:${AWS::AccountId}:table/*/backup/*'), 
```

Comment:
Are these correct? Looking at https://docs.aws.amazon.com/service-authorization/latest/reference/list_awsbackup.html#awsbackup-backupVault, the arn should be something like `arn:${Partition}:backup:${Region}:${Account}:backup-vault:${BackupVaultName}` for a vault, `arn:${Partition}:backup:${Region}:${Account}:backup-plan:${BackupPlanId}` for a plan. I'm also not entirely sure you need that third resource listed since none of the actions likely map to a table resource?

Something else to note is we generally want to reduce the permissions as much as possible. For pipelines in particular, we try to limit resources that it can change to those within the same namespace that the pipeline will deploy to. You can only do this if: 
1. The iam policy allows for writing resource level permissions based on name
2. The naming of that resource is handled by cloudformation, or you explicitly name the resource using the namespace prop

In this case, it looks like you can do this for vaults, since it uses BackupVaultName. But, it does not look like you can do this for backup plan, since it's by plan id which is unknown at synth time.  So for example, the vault resource would be better scoped with something like:
`cdk.Fn.sub('arn:aws:backup:${AWS::Region}:${AWS::AccountId}:backup-vault:' + namespace + '*')`
You may also need to change your backup plan name to use namespace, ex change https://github.com/ndlib/marble-blueprints/pull/321/files#diff-c50cd9e75848a4c8f39b258ebb285fcede0e568f404f4503a35addfb2a51e5adR253 to:
```
const plan = backup.BackupPlan.dailyMonthly1YearRetention(this, `${props.namespace}DynamoDbBackupPlan`)
```